fedocal/fedocallib/__init__.py

```diff
@@ -722,6 +737,8 @@ def add_meetings_to_vcal(ical, meetings):
         be added.
     :arg meetings: a list of fedocal.model.Meeting object to convert to
         iCal and add to the provided calendar.
+    :kwarg enable_reminder: a boolean telling whether to add reminder
+        for the meetings in the iCal object or not.
```

Comment:
Currently, it's not being used from the UI. But, it allows to add reminders for each meeting in a list of meetings. Oh! yes, I forgot the ``reminders`` param. Anyways, we are gonna replace ``enable_reminder`` with ``reminders`` param.